test/modules/implicit/parseTwice/main.good

```diff
@@ -1,2 +1,3 @@
+./SomeLibrary.chpl:1: error: In use/imported file, module name M does not match file name SomeLibrary.chpl
```

Comment:
* Seems like this should say "used/imported"
* "file name" -> "filename"
* I'd single-quote 'M' and 'SomeLibrary.chpl'

Reply:
First time through, I missed that this was an error rather than a warning.  I think it should be a warning as explained in https://github.com/chapel-lang/chapel/issues/21702#issuecomment-1447436684, especially since we're about to get an error due to not being able to find a module named `SomeLibrary`.